Source/FieldSolver/SpectralSolver/SpectralSolverRZ.H

```diff
@@ -112,10 +116,53 @@ class SpectralSolverRZ
          */
         void VayDeposition (const int lev, std::array<std::unique_ptr<amrex::MultiFab>,3>& current);
 
+        /**
+         * \brief Copy spectral data from component \c src_comp to component \c dest_comp
+         *        of \c field_data.fields.
+         *
+         * \param[in] src_comp  component of the source FabArray from which the data are copied
+         * \param[in] dest_comp component of the destination FabArray where the data are copied
+         */
+        void CopySpectralDataComp (const int src_comp, const int dest_comp)
+        {
+            // The last two arguments represent the number of components and
+            // the number of ghost cells to perform this operation
+            const int ncomps = m_n_rz_azimuthal_modes*2 - 1;
+            Copy(field_data.fields, field_data.fields, src_comp, dest_comp, ncomps, 0);
+        }
```

Comment:
@dpgrote I have a question, since I'm debugging this PR. Does this implementation highlighted here (same for the other new functions below) seem to you to be the correct way to replicate what is done in the Cartesian case (see code linked)?
https://github.com/ECP-WarpX/WarpX/blob/f3ced1388b634eacbd4b1d64bd98be4ab444d6e5/Source/FieldSolver/SpectralSolver/SpectralSolver.H#L157-L162

Reply:
Thinking about this, I think it would be more appropriate that the call to `Copy` should be done in the `SpectralFieldDataRZ` class, following the pattern with the transforms. So the call here in the `SpectralSolverRZ` class, would be `field_data.CopySpectralDataComp(src_comp, dest_comp);`. Then the method `CopySpectralDataComp` would be added to `SpectralFieldDataRZ` which makes the call to `Copy`. The number of modes is only saved in `SpectralFieldDataRZ` which knows about the details of the `field` object. Also, this should be `ncomps = n_rz_azimuthal_modes`. The transformed data is complex, combining the real and imaginary components, so the number of MultiFab components is equal to the number of modes. Thanks for adding this to the RZ solver.

Reply:
Thank you for the feedback, Dave. I made the change in https://github.com/ECP-WarpX/WarpX/pull/2111/commits/d4daa40ddd7999fa9354bc1dd35c7b9ac149e6ce. Please let me know if this matches what you suggested.